lib/jobs/upsert-node-job.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+module.exports = UpsertNodeAttributeFactory;
+
+var di = require('di');
+
+di.annotate(UpsertNodeAttributeFactory, new di.Provide('Job.Upsert.Node.Attribute'));
+di.annotate(UpsertNodeAttributeFactory, new di.Inject(
+    'Job.Base',
+    'Util',
+    'Services.Waterline',
+    'Logger'
+    )
+);
+
+function UpsertNodeAttributeFactory(BaseJob, util, waterline, Logger) {
+    var logger = Logger.initialize(UpsertNodeAttributeFactory);
+
+    function UpsertNodeAttribute(options, context, taskId) {
+        UpsertNodeAttribute.super_.call(this, logger, options, context, taskId);
+        this.nodeId = this.options.target;
```

Comment:
This is a use case isssue, this is an interesting feature, we can execute logic with parameter from url. @benbp how do you think?